<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QDialog" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dashboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>選擇項目</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="verticalGroupBox">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="data_base_comboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ListButton">
          <property name="text">
           <string>List</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>統計項</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="5" column="2">
       <widget class="QComboBox" name="EndYearcomboBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="element1_comboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="element1_label">
        <property name="text">
         <string>指標1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="StartYearcomboBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="element2_label">
        <property name="text">
         <string>指標2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="StartYear_label">
        <property name="text">
         <string>統計開始年分</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="element2_comboBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="EndYearlabel">
        <property name="text">
         <string>統計結束年分</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="element3_comboBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="element3_label">
        <property name="text">
         <string>指標3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="dowload">
     <property name="text">
      <string>載入/下載</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="InputNameLabel">
     <property name="text">
      <string>檔案名稱:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="InputFileLabel">
     <property name="text">
      <string>檔案位置:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="verticalGroupBox">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="display"/>
        </item>
        <item>
         <layout class="QGridLayout" name="chart_select">
          <item row="0" column="2">
           <widget class="QRadioButton" name="point_chart">
            <property name="text">
             <string>點圖</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="Pie_Chart">
            <property name="text">
             <string>圓餅圖</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="Bar_chart">
            <property name="text">
             <string>直方圖</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="Line_chart">
            <property name="text">
             <string>折線圖</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
